test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -109,5 +109,130 @@ defmodule Membrane.RTC.HLSEndpointTest do
+      audio_track_id = "test-audio-track-id"
+      stream_id = "test-stream-2"
+
+      track =
+        Engine.Track.new(
+          :video,
+          stream_id,
+          video_file_endpoint_id,
+          :H264,
+          nil,
+          [:raw],
+          nil,
+          id: video_track_id
+        )
+
+      hls_endpoint = %HLS{
+        rtc_engine: rtc_engine,
+        owner: self(),
+        output_directory: Path.join(["./", "test", "hls_output"]),
+        target_window_duration: :infinity,
+        framerate: {60, 1}
+      }
+
+      :ok = Engine.add_endpoint(rtc_engine, hls_endpoint, endpoint_id: hls_endpoint_id)
+
+      parser = %Membrane.H264.FFmpeg.Parser{
+        attach_nalus?: true,
+        skip_until_parameters?: false,
+        framerate: {60, 1}
+      }
+
+      file_endpoint = %FileEndpoint{
+        rtc_engine: rtc_engine,
+        file_path: video_file_path,
+        track: track,
+        interceptor: fn link_builder ->
+          Membrane.ParentSpec.to(link_builder, :parser, parser)
+        end
+      }
+
+      audio_track =
+        Engine.Track.new(
+          :audio,
+          stream_id,
+          audio_file_endpoint_id,
+          :AAC,
+          nil,
+          [:raw],
+          nil,
+          id: audio_track_id
+        )
+
+      test_file_endpoint = %FileEndpoint{
+        rtc_engine: rtc_engine,
+        file_path: Path.join(@fixtures_dir, "audio.aac"),
+        track: audio_track,
+        interceptor: fn link_builder ->
+          Membrane.ParentSpec.to(link_builder, :parser, %Membrane.AAC.Parser{
+            out_encapsulation: :none
+          })
+        end
+      }
+
+      :ok = Engine.add_endpoint(rtc_engine, file_endpoint, endpoint_id: video_file_endpoint_id)
+
+      assert_receive %Message.MediaEvent{rtc_engine: ^rtc_engine, to: :broadcast, data: data}
+
+      assert %{
+               "type" => "tracksAdded",
+               "data" => %{
+                 "trackIdToMetadata" => %{video_track_id => nil},
+                 "peerId" => video_file_endpoint_id
+               }
+             } == Jason.decode!(data)
+
+      :ok =
+        Engine.add_endpoint(rtc_engine, test_file_endpoint, endpoint_id: audio_file_endpoint_id)
+
+      assert_receive %Message.MediaEvent{rtc_engine: ^rtc_engine, to: :broadcast, data: data}
+
+      assert %{
+               "type" => "tracksAdded",
+               "data" => %{
+                 "trackIdToMetadata" => %{audio_track_id => nil},
+                 "peerId" => audio_file_endpoint_id
+               }
+             } == Jason.decode!(data)
+
+      Engine.message_endpoint(rtc_engine, video_file_endpoint_id, :start)
+      Engine.message_endpoint(rtc_engine, audio_file_endpoint_id, :start)
+
+      assert_receive({:playlist_playable, :video, ^stream_id, ^video_file_endpoint_id}, 5_000)
+      assert_receive({:playlist_playable, :audio, ^stream_id, ^audio_file_endpoint_id}, 5_000)
+
+      Process.sleep(10_000)
```

Comment:
The test file seems to have one second. Why do we need to wait for 10 seconds here?